.github/workflows/ci.yml

```diff
@@ -16,7 +16,7 @@ jobs:
       subcommands: |
         python3 scripts/download_wasm.py
         npm install
-      kms-version: 4.5.0
+      kms-version: feature-pki
```

Comment:
as a memo, before merging this PR, all `feature-pki` must be replaced by `4.6.0` as soon the KMS v4.6.0 is released